src/core/caniuse.js

```diff
@@ -103,60 +108,69 @@ function normalizeConf(conf) {
  * promises to get content for canIUse table
  * @param {Object} conf              normalized respecConfig.caniuse
  */
-async function canIUse(conf, cache) {
-  return new Promise(async (resolve) => {
+async function fetchAndCacheJson(conf) {
+  const cache = new IDBCache("respec-caniuse", ["caniuse"]);
+
+  const url = conf.apiURL ?
+    conf.apiURL.replace("{FEATURE}", conf.feature)
+    : `https://raw.githubusercontent.com/Fyrd/caniuse/master/features-json/${conf.feature}.json`;
+
+  return new Promise(async (resolve, reject) => {
```

Comment:
I'm not sure we need, `return new Promise(async (resolve, reject) => ` here... looking at the code, looks like we can  naturally return or throw.

Reply:
why did i fail to notice that! On it!